repair/row_level.hh

```diff
@@ -279,5 +279,5 @@ future<> repair_cf_range_row_level(repair::shard_repair_task_impl& shard_task,
         service::frozen_topology_guard topo_guard);
 future<std::list<repair_row>> to_repair_rows_list(repair_rows_on_wire rows,
         schema_ptr s, uint64_t seed, repair_master is_master,
-        reader_permit permit, repair_hasher hasher);
+        reader_permit permit, repair_hasher hasher, bool msg_split_feature = false);
```

Comment:
Better to avoid the default parameter, so we have to change all the call sites.

Reply:
OK.  I will drop the flag in the new version though.